cli/src/main/java/org/aya/cli/repl/ReplConfig.java

```diff
@@ -36,11 +35,10 @@ public class ReplConfig implements AutoCloseable {
    */
   public @UnknownNullability RenderOptions renderOptions = new RenderOptions();
   public transient @NotNull StringStylist stylist;
```

Comment:
Maybe `private` or remove the useless `getStylist()`

Reply:
Actually we do not need to store the stylist here. But now it behaves a little bit different: it parses the vsc file every time we render a Doc. I am not sure if it a big performance issue....

Reply:
We may use a LazyValue with expiration check to store the stylist if so.

Reply:
> Actually we do not need to store the stylist here. But now it behaves a little bit different: it parses the vsc file every time we render a Doc. I am not sure if it a big performance issue....

I think the `stylist` should be memorized.

Reply:
Then it should be stored in RenderOptions itself.

I am fixing it tomorrow.

Reply:
You are fixing it in the next PR